internal/cmd/commands/server/worker_shutdown_reload_test.go

```diff
@@ -81,23 +81,23 @@ func TestServer_ShutdownWorker(t *testing.T) {
 	tcl := targets.NewClient(client)
 	tgtL, err := tcl.List(ctx, scope.Global.String(), targets.WithRecursive(true))
 	require.NoError(err)
-	require.Len(tgtL.Items, 2)
+	require.LessOrEqual(2, len(tgtL.Items))
 	tgt := tgtL.Items[0]
 	require.NotNil(tgt)
 	require.NotNil(tgtL.GetItems()[1])
 
 	// Create test server, update default port on target
 	ts := helper.NewTestTcpServer(t)
 	require.NotNil(ts)
-	defer ts.Close()
+	t.Cleanup(ts.Close)
 	tgtR, err := tcl.Update(ctx, tgt.Id, tgt.Version, targets.WithTcpTargetDefaultPort(ts.Port()))
 	require.NoError(err)
 	require.NotNil(tgtR)
 
 	// Authorize and connect
 	// This prevents us from running tests in parallel.
 	tg.SetupSuiteTargetFilters(t)
-	sess := helper.NewTestSession(ctx, t, tcl, tgt.Id)
+	sess := helper.NewTestSession(ctx, t, tcl, tgt.Id, helper.WithSkipSessionTeardown(true))
```

Comment:
It's not a flake -- or at least, that's not what this is for. This test purposefully shuts down the worker prior to test cleanup. As a result since the new code used in the tests (which is the API code) tries to send a session teardown request to the worker (without this flag) then we will always receive an error when running this test. So this flag is indicating that we do not want to send this (because it is expected to return an error, and we do not want that).